Sources/StreamChatUI/Components.swift

```diff
@@ -94,6 +94,9 @@ public struct Components {
     /// The view that shows a playing video.
     public var playerView: PlayerView.Type = PlayerView.self
 
+    // The view that displays unread messages count for messages and thread replies
```

Comment:
Is this view also presentable inside a thread? My understanding is that it's not. If that's right, shall we update the comment?